usl_pipeline/usl_lib/usl_lib/shared/suppliers.py

```diff
@@ -0,0 +1,49 @@
+import abc
+import os
+
+
+class ChunkFilePathGenerator(abc.ABC):
```

Comment:
Do we anticipate additional implementations of this interface? If not, do we need to define the interface?

Reply:
In theory the caller might want to organize chunks in some different way (for instance to place all the layers corresponding to one chunk position to its own folder). But I see that this whole amount of flexibility isn't comfortable for you. Removed it all together.